util/dvsim/FlowCfg.py

```diff
@@ -403,43 +403,46 @@ def gen_results(self, results):
         '''
         for item in self.cfgs:
             result = item._gen_results(results)
-            item.write_results_html()
+            item.write_results_html("results.html", item.results_md)
             log.info("[results]: [%s]:\n%s\n", item.name, result)
             log.info("[scratch_path]: [%s] [%s]", item.name, item.scratch_path)
             log.log(VERBOSE, "[results_path]: [%s] [%s]", item.name, item.results_path)
             self.errors_seen |= item.errors_seen
 
         if self.is_primary_cfg:
             self.gen_results_summary()
-            self.write_results_html()
+            self.write_results_html("summary.html", item.results_summary_md)
         self.gen_email_html_summary()
 
     def gen_results_summary(self):
         '''Public facing API to generate summary results for each IP/cfg file
         '''
         return
 
-    def write_results_html(self):
+    def write_results_html(self, results_name, results_md):
```

Comment:
How about:
```suggestion
    def write_results_html(self, filename, text_md):
```

Reply:
Thank you Sri for the suggestion. This generic name is better.